backend/vercel/vercel_test.go

```diff
@@ -0,0 +1,55 @@
+package vercel
+
+import (
+	"encoding/json"
+	"github.com/openlyinc/pointy"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+type MockHTTP struct {
+}
+
+func (c *MockHTTP) Do(req *http.Request) (*http.Response, error) {
+	return &http.Response{}, nil
```

Comment:
Is this stubbing the response from vercel?

Reply:
whoops, this from a different attempt at writing the test when i was trying to mock. cleaning this up